src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1296,16 +1296,15 @@ class TcgStatics {
     }
   }
   static void opponentCantPlaySupporterNextTurn(def _delegate){
+    bc "${_delegate.thisMove} - Supporters can't be played from ${opp.owner.getPlayerUsername(bg)}'s hand during theit next turn"
     _delegate.delayed {
       before PLAY_TRAINER, {
         if(ef.supporter && bg.currentTurn==_delegate.self.owner.opposite) {
-          wcu "Bother prevents playing supporters"
+          wcu "${_delegate.thisMove} prevents playing supporters"
           prevent()
         }
       }
       unregisterAfter 2
-      after SWITCH, _delegate.self, {unregister()}
-      after EVOLVE, _delegate.self, {unregister()}
```

Comment:
Why did you remove these?

Reply:
From the PR's OP:
> It also removes the unregister conditions. From what I see, neither of those attacks nor Bother should be removed if the attacking Pokémon is switched or evolved. Putting this as a PR since it touches a static method, not sure if it may affect some legacy card I'm not aware of.

Main reason I put it as a PR, besides touching a static. Is there some card that needs this when calling this static? None of the ones in the repo seem to.

Reply:
The way the relevant attacks are worded, these should effects on the Player, not the Pokémon. Switching and evolving should have nothing to do with the effect.